src/main/java/nl/hsac/fitnesse/fixture/util/JsonHelper.java

```diff
@@ -25,12 +28,13 @@
     public String format(String json) {
         String result = null;
         if (json != null){
+            Gson gson = new GsonBuilder().setPrettyPrinting().create();
+            JsonParser jsonParser = new JsonParser();
+
             if (json.startsWith("{")) {
-                result = new JSONObject(json).toString(4);
+                return gson.toJson(jsonParser.parse(json).getAsJsonObject());
```

Comment:
Please don't return from `if` branch and reduce code duplication between branches